web/test/helpers/helpers-formation.tsx

```diff
@@ -107,9 +106,9 @@ export const preparePage = ({
           formationFormData: createEmptyFormationFormData(),
         }),
   });
+
   const internalMunicipalities = [
-    profileData?.municipality ?? generateMunicipality({ displayName: "GenericTown" }),
-    ...(municipalities ?? []),
```

Comment:
pebble - why are we removing the ability to pass in muni?

Reply:
Please see previous comment in `helpers-formation.tsx`.